client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -14,14 +16,24 @@ import FormattedHeader from 'components/formatted-header';
 import Main from 'components/main';
 import PaginationFlow from './pagination-flow';
 import redirectNonJetpackToGeneral from 'my-sites/site-settings/redirect-to-general';
+import ReturnToPreviousPage from 'my-sites/site-settings/render-return-button/back';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
 
 class DisconnectSite extends Component {
+	// when complete, the flow will start from /settings/manage-connection
+	handleClickBack = () => {
+		const { siteSlug } = this.props;
+
+		page( '/settings/manage-connection/' + siteSlug );
+	};
+
 	render() {
 		const { translate } = this.props;
 
 		return (
 			<Main className="disconnect-site site-settings">
 				<DocumentHead title={ translate( 'Site Settings' ) } />
+				<ReturnToPreviousPage onBackClick={ this.handleClickBack } { ...this.props } />
```

Comment:
The button should be on the very top left-hand corner of the view area, so maybe we need to pop it outside of the `Main` area and create an outer div.

Reply:
Same level as the '<- Switch Site', correct?

Reply:
Yes, kind of like this:

![image](https://user-images.githubusercontent.com/390760/29970690-6b492342-8f1d-11e7-9d5b-6c02b0649893.png)


We could even make the case that it's better to drop the sidebar altogether for this survey. What do you think @rickybanister ?

Reply:
I agree with that. The sidebar only really provides a distraction here. Full screen will help us accomplish our goal better and will help us improve the mobile view more.